packages/web/.env

```diff
@@ -12,3 +12,5 @@ NEXT_PUBLIC_FEATURE_PP_PAGE=true
 # API
 NEXT_PUBLIC_GRAPH_URL=http://localhost:4000
 
+# Monitoring
+NEXT_PUBLIC_LOGROCKET_APP_ID=
```

Comment:
Maybe better move to `.env.example`?

Reply:
hm, tbh, don't see a reason to create a new file just for this